src/client/components/Header/index.js

```diff
@@ -3,11 +3,19 @@ import React from 'react';
 import './header.css';
 import Title from '../Title';
 import LanguageSelector from '../LanguageSelector';
+import HeaderLeft from '../HeaderLeft';
+import HeaderRight from '../HeaderRight';
+import SidePanelButton from '../SidePanelButton';
 
 const Header = () => (
```

Comment:
Header should not include HeaderLeft and HeaderRight inside it's definition but ouside within App